pkg/backend/aws/ecs.go

```diff
@@ -369,6 +360,33 @@ func (ab *Backend) getLogEventsForTask(
 		return errors.Wrap(err, "could not describe task definition")
 	}
 
+	ch := make(chan *ecs.DescribeTasksOutput, 1)
+	go func() {
+		counter := 0
+		for {
```

Comment:
```suggestion
		for now := range time.Tick(time.Minute) {
```

then you can remove your sleep on line 371